src/onegov/org/views/directory.py

```diff
@@ -565,10 +565,27 @@ def handle_change_request(self, request, form, layout=None):
     template='directory_entry.pt')
 def view_directory_entry(self, request, layout=None):
 
+    directory = self.directory
+
+    siblings = request.exclude_invisible(ExtendedDirectoryEntryCollection(
+        directory,
+        published_only=not request.is_manager
+    ).query())
+
+    entry_index = siblings.index(self)
+
+    prev_entry = siblings[entry_index - 1] if entry_index != 0 else False
+    next_entry = siblings[
+        entry_index + 1] if entry_index != len(siblings) - 1 else False
+    more_entries = bool(prev_entry or next_entry)
```

Comment:
Should be `prev_entry is not False or next_entry is not False`, because prev_entry can be 0 (which converts to False)

Reply:
But prev_entry will contain the entry itself (if there is one), not the index. How can it be 0?